docs/source/API/core/initialize_finalize/initialize.rst

```diff
@@ -0,0 +1,91 @@
+``initialize``
+==============
+
+.. role::cpp(code)
+    :language: cpp
+
+Defined in header ``<Kokkos_Core.hpp>``
+
+Usage: 
+
+.. code-block:: cpp
+
+    Kokkos::initialize(argc, argv);
+    Kokkos::initialize(Kokkos::InitializationSettings()  // (since 3.7)
+                           .set_disable_warnings(true)
+                           .set_num_threads(8)
+                           .set_map_device_id_by("random"));
+
+Initializes the Kokkos execution environment.
+This function must be called before any other Kokkos API functions or
+constructors.  There are a small number of exceptions, such as
+``Kokkos::is_initialized`` and ``Kokkos::is_finalized``.
```

Comment:
Content suggestion:

I assume they return a Boolean?  Please also indicate the return types, and perhaps point to / mention the example below.

Reply:
we can put links to the pages documenting these two functions

Reply:
I can't find the documentation associated with these 2 functions, where are they? I'm not sure if they already exist

Reply:
i don't think there are documented, and i think `is_initialized` should be documented on this page directly since it related to the initilization. For now, let's leave them like this #232